examples/src/bin/tlsserver-mio.rs

```diff
@@ -3,8 +3,16 @@ use std::sync::Arc;
 use mio::net::{TcpListener, TcpStream};
 
 #[macro_use]
+#[cfg(feature = "logging")]
 extern crate log;
 
+#[cfg(not(feature = "logging"))]
+#[macro_use]
+mod log {
+    macro_rules! debug    ( ($($tt:tt)*) => {{}} );
+    macro_rules! error    ( ($($tt:tt)*) => {{}} );
+}
```

Comment:
If folks dislike duplicating these macro rules we could also export them from the rustls lib.rs - I had that in an earlier version but decided a little duplication was more appetizing than additional API surface.